CHANGELOG.md

```diff
@@ -1,13 +1,26 @@
 # Changelog
 
-## [Unreleased]
+## [0.17.0]
 
 ### Added
 
 - Added an experimental swagger definition that includes the specification for
   the vsock API call.
 - Added a signal handler for `SIGBUS` and `SIGSEGV` that immediately terminates
   the process upon intercepting the signal.
+- Added documentation for signal handling utilities.
+- Added [alpha] aarch64 support.
+- Added metrics for read and written bytes by the MMDS, Net and Block Devices.
+
+### Changed
+
+- Unmount and mount events propagate into the chroot location of the jailer.
+- Disallow invalid seccomp levels by exiting with error.
+
+### Fixed
+
```

Comment:
We also need an entry for this commit bfaf26c7f68fec9fdccbd7ef65971a280d28a91f

Reply:
yes, I had put it in "Changed section", first entry (Unmount and mount...). I moved it to Fixed and tried to find a better explanation.